src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1650,11 +1649,43 @@ format layout_optimizer::get_preferred_format(program_node& node) {
         if (input_layout.format.dimension() == 5 &&
             (input_layout.data_type == data_types::f32 || input_layout.data_type == data_types::f16))
             expected = format::bfzyx;
+    } else if (node.is_type<resample>()) {
+        // if the resample is in the last part of the network and there are no users using blocked format,
+        // it is better to reorder to bfyx before resample is done.
+        if (all_users_simple_format(node, node, 0, 10)) {
+            const auto& dim = format::dimension(node.get_output_layout().format);
```

Comment:
I think you can use `get_default_format()` function here

Reply:
get_default_format is a static function in layout.cpp, so cannot be used here :( I think we can move this to more common header

Reply:
Updated